libs/ui/src/components/NeoModal/NeoModal.vue

```diff
@@ -6,7 +6,7 @@
     :destroy-on-hide="destroyOnHide"
     :can-cancel="canCancel"
     :full-screen="fullScreen"
-    :content-class="[contentClass, noShadow ? 'no-shadow' : '']"
+    :content-class="[...contentClassName, noShadow ? 'no-shadow' : '']"
```

Comment:
```suggestion
    :content-class="[...contentClassName, { 'no-shadow': noShadow }]"
```

Reply:
I tested it but `o-modal` component does not accept this way.